### ISSUE TITLE
Any Updates on Reverse Engineering Apple SandBox?

### ISSUE THREAD

**Issue by ghost** [steven-michaud/reverse-engineering-on-osx#1]
Sorry this is not really an issue but I am writing a thesis on sandboxing in OS X, would like to share your progress maybe?

**Comment by steven-michaud** [comment]
Interesting!  I'm working on a project called SandboxMirror.  It basically tracks all occasions on which Apple's sandbox might constrain a given process's behavior, and allows you to log some or all of this behavior.  When I finish I'll post it on GitHub.  Since I'm almost done, that's probably only a few weeks away.

I started working on SandboxMirror while I was still working at Mozilla (I retired about 6 months ago).  I opened a couple of related bugs in Mozilla's bug reporting system, which you might want to take a look at:

https://bugzilla.mozilla.org/show_bug.cgi?id=1186187
https://bugzilla.mozilla.org/show_bug.cgi?id=1185084

But you'll find much more information in my SandboxMirror project, once I release it -- both by using it and by looking at its source code.  Once I do release it, I'll want to hear from people like you how I might make it more useful.
